MAUI/DataForm/CustomDataFormItem.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: Working with DataForm in Maui DataForm control | Syncfusion
+description: Learn here all about Working with DataForm support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+## Adding custom DataFormItems
+
+Support has been provided to generate custom DataFormItems for the defined business model using the [Items]`Link` property of the `SfDataForm` class. You need to set the [AutoGenerateItems](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_AutoGenerateItems) property to false to restrict the auto generation of DataFormItems. 
```

Comment:
I can see xamarin forms properties AutoGenerateItems and link, don't copy xamarin content as it is

Reply:
xamarin link removed